h2o-py/h2o/explanation/_explain.py

```diff
@@ -1230,6 +1230,49 @@ def pd_multi_plot(
 
 def _center(col):
     col[:] = col - col[0]
+def _prepare_grouping_frames(frame, grouping_variable):
+    if (grouping_variable not in frame.names):
+        raise ValueError("Grouping variable '" + grouping_variable + "' is not present in frame!")
+    if (not frame[grouping_variable].isfactor()[0]):
+        raise ValueError("Grouping variable has to be categorical!")
+    categories = frame[grouping_variable].categories()
+    if len(categories) > 10:
+        raise ValueError("Grouping variable option is supported only for variables with 10 or fewer levels!")
+    frames = list()
+    for curr_category in categories:
+        key = "tmp_" + curr_category + str(random.randint(0,1000))
+        expr = "(tmp= " + key + "(rows " + frame.frame_id + " (==(cols " + frame.frame_id + " [" + str(frame.names.index(grouping_variable)) + "] ) '" + curr_category + "') ))"
+        h2o.rapids(expr)
+        frames.append(h2o.get_frame(key))
+    return frames
+
+def _handle_grouping(frame, grouping_variable, save_plot_path, model, column, target, max_levels, figsize, colormap):
+    frames = _prepare_grouping_frames(frame, grouping_variable)
+    result = list()
+    if save_plot_path is not None:
+        len_path = len(save_plot_path)
+    i = 0
+    for curr_frame in frames:
```

Comment:
nit/optional: It's fine to use explicit counter, but a little more pythonic would be to use `enumerate` this way
```python
for idx, curr_frame in enumerate(frames):
    ...
```
and you can spare the counter